pontoon/base/static/js/translate.js

```diff
@@ -107,7 +107,7 @@ var Pontoon = (function (my) {
                       (this.approved ? this.approved_user ?
                         'Approved by ' + this.approved_user : '' : 'Approve') +
                       '"></button>' +
-                      ((self.user.email && (self.user.email === this.email)) ? '<button class="delete fa" title="Delete"></button>' : '') +
+                      ((self.user.email && (self.user.email === this.email) || self.user.isLocaleManager) ? '<button class="delete fa" title="Delete"></button>' : '') +
```

Comment:
Done, could you look again?

On Thu, Dec 17, 2015 at 2:48 PM, Matjaž Horvat <notifications@github.com>
wrote:

> In pontoon/base/static/js/translate.js
> <https://github.com/mozilla/pontoon/pull/308#discussion_r47906607>:
>
> > @@ -107,7 +107,7 @@ var Pontoon = (function (my) {
> >                        (this.approved ? this.approved_user ?
> >                          'Approved by ' + this.approved_user : '' : 'Approve') +
> >                        '"></button>' +
> > -                      ((self.user.email && (self.user.email === this.email)) ? '<button class="delete fa" title="Delete"></button>' : '') +
> > +                      ((self.user.email && (self.user.email === this.email) || self.user.isLocaleManager) ? '<button class="delete fa" title="Delete"></button>' : '') +
>
> And that means this is all we need to change in this PR. You can revert
> other changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/pontoon/pull/308/files#r47906607>.
>



-- 
Kind regards

​Jarek "jotes" Śmiejczak,
​P​
ython programmer​ by day, mozillian by heart..
Homepage: http://joyfulhack.in, Github: http://github.com/jotes
Mobile: +48693027040, Mozillian: http://mozillians.org/u/jotes